test/WebSites/FiltersWebSite/Controllers/ResourceFilterController.cs

```diff
@@ -31,7 +33,9 @@ private class ShortCircuitWithFormatterAttribute : Attribute, IResourceFilter
 
             public ShortCircuitWithFormatterAttribute()
             {
-                _formatters = new IOutputFormatter[] { new JsonOutputFormatter() };
+                _formatters = new IOutputFormatter[] { new JsonOutputFormatter(
+                    new JsonSerializerSettings(),
```

Comment:
Hmm, should I do the Right Thing<sup>:tm:</sup> and get the settings from `IOptoins<MvcJsonOptions>`? That is, is it :ok: to demonstrate the lazy approach in one of our web sites? Same question for `JsonFormatterController` below.